src/components/AssetsComponent.jsx

```diff
@@ -102,50 +98,25 @@ export default class AssetsComponent extends Component {
           <div id="page-heading-section">
             <Header as="h1" id="page-headings" floated="left" content={contentTitle} />
             <Divider id="assets-divider" />
-            {showFilter && (
-              <FilterButton
-                activePage={this.props.activePage}
-                limit={this.state.limit}
-                selected={this.props.selected}
-                filterAction={this.props.getAssetsAction}
-                disabled={this.props.isLoading}
-              >
-                <React.Fragment>
-                  <FilterComponent
-                    index={0}
-                    option={this.props.filterData[0]}
-                    selected={this.props.selected}
-                    filterSelection={this.props.filterSelection}
-                  />
-
-                  <FilterComponent
-                    index={1}
-                    option={this.props.filterData[1]}
-                    selected={this.props.selected}
-                    filterSelection={this.props.filterSelection}
-                  />
-                </React.Fragment>
-              </FilterButton>
-            )}
           </div>
-          <AssetsTableContent
+          <AssetsTabComponent
             activePage={this.props.activePage}
-            assets={this.props.assetsList[currentAssets] || assets}
+            limit={this.state.limit}

```

Comment:
@Thuku @Ewanjiru if you read the guideline I'm working on, the number of props being passed to a child component in a render function are limited to a maximum of 8. You are passing more than this number to `AssetsTabComponent`. You need to rethink the structure of your component in order to limit the props to 8

Reply:
Noted.